modules/mobile/app/models/mobile/v0/user_accessible_services.rb

```diff
@@ -21,7 +21,7 @@ def available
       def service_auth_map
         @service_auth_map ||= {
           appeals: access?(appeals: :access?),
-          appointments: access?(vaos: :access?) && @user.icn.present?,
+          appointments: access?(vaos: :access?) && @user.icn.present? && access?(vaos: :facilities_access?),
```

Comment:
I think this will break the intended logic. The intention for that facilities access method was to return a specific error message that FE can key off of and show a banner about how they don't have facilities. If this returns false, then FE won't get a chance to get that error message.

Reply:
Was this choice discussed somewhere? The original intention behind the vets-api policy update was so the mobile authed services checks act more accurately and so that we don't make calls to apis we know will fail.

Reply:
So we want to allow access even though it's doomed to fail? Is that better than just not letting them have access at all?

Reply:
The driving factor behind adding this check was when the FE added appts overview to the home screen it massively spiked the number of calls being made to VAOS and adding the facilities check, which web has on the FE, to the vets-api policy (and our authed services by proxy) would lower the number of calls being made. If there was a discussion I'm not aware of where mobile FE/UX decided they wanted to specific isolate this exact reason for failure then we can let them through but it does go against our pattern a bit and also goes against the purpose of the policy updating epic.

Reply:
Okay cool. Let me take a look at those threads real quick. It's likely not an issue to let them through. You're correct that it stops upstream calls still, but generally the purpose of the authed services endpoint is to stop calls from being made to vets-api unnecessarily

Reply:
So looking at those threads it looks like no actual decision was made for if we want a banner or not or what the banner should say. @Tonksthebear go ahead and remove this for now, sorry for the extra headache. I'll do some circling on the banner and depending on what comes out of it we might end up adding this back.